app/models/group.rb

```diff
@@ -148,6 +148,14 @@ def resort?
     Group.resorts.include?(self)
   end
 
+  def primary_memberships
+    primary_membership_ids = User.joins(:primary_membership).where('memberships.group_id': id)
+                                 .where.not(svie_member_type: SvieUser::NOT_MEMBER)
+                                 .select('memberships.id as primary_membership_id')
+                                 .map(&:primary_membership_id)
```

Comment:
I think pluck should work on the AR query object, so you don't have to iterate over it.

```suggestion
                                 .pluck(:primary_membership_id)
```

Reply:
Thank you for mentioning, I always forget about the power of pluck! :rocket: